pandas/core/indexes/datetimes.py

```diff
@@ -1904,14 +1904,17 @@ def delete(self, loc):
 
     def tz_convert(self, tz):
         """
-        Convert tz-aware DatetimeIndex from one time zone to another (using
-        pytz/dateutil)
+        Convert tz-aware DatetimeIndex from one time zone to another.
+
+        When using DatetimeIndex providing with timezone this method
+        converts tz(timezone)-aware DatetimeIndex from one timezone
+        to another using pytz/dateutil.
```

Comment:
you can remove the pytz/dateutil part here

Reply:
let's coordinate the text in this doc-string with #20050 which is sister operation.

Reply:
I have removed the pytz/dateutil  part